systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -43,6 +43,18 @@
     String STRIMZI_DEPLOYMENT_NAME = "strimzi-cluster-operator";
     String IMAGE_PULL_POLICY = "Always";
 
+    /**
+     * Kafka Bridge JSON encoding with JSON embedded format
+     */
+    String KAFKA_BRIDGE_JSON_JSON = "application/vnd.kafka.json.v2+json";
```

Comment:
the one with BINARY embedded format is missing.

Reply:
@Frawless I guess Paolo talking about `"application/vnd.kafka.binary.v2+json"`